.changeset/dry-bananas-ring.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-notifications-backend': minor
+'@backstage/plugin-notifications': minor
```

Comment:
```suggestion
'@backstage/plugin-notifications-backend': patch
'@backstage/plugin-notifications': patch
```

Switching these over to `patch` since there are no breaking changes. We use `minor` to mark breaking changes while packages are at `0.x`